### ISSUE TITLE
Topic: Title input text field doesn't expand

### ISSUE THREAD

**Issue by BeccaMelhuish** [citizenos/citizenos-fe#646]
When user wrote a Title that was longer than one line in the text field, it went off the right-hand edge of the field, out of view as so (should say "community"): 

![image](https://github.com/citizenos/citizenos-fe/assets/61320526/05407296-a2ff-4b3c-bad4-370c3a84c4ff)

**Comment by ilmartyrk** [comment]
@BeccaMelhuish @anettlinno  it is an input field and user can edit the title, also see the text that is out of view, not sure if it is really an issue in that sense. I could change the input to textarea and that would allow breaking the line, but it might also cause other styling issues, so I would rather close this one with no further action

**Comment by ssin1901** [comment]
@ilmartyrk @BeccaMelhuish I can see what you mean in terms of even a short title going off the screen and it looking a bit odd. I have included screenshot1 from Desktop and screenshot2 from Mobile view.

<img width="922" alt="Screenshot 2024-02-19 at 15 37 27" src="https://github.com/citizenos/citizenos-fe/assets/72747107/92615ecb-b45e-4875-9d77-50189a5f5a0c">

<img width="425" alt="Screenshot 2024-02-19 at 15 38 53" src="https://github.com/citizenos/citizenos-fe/assets/72747107/6a0834e4-cf43-4338-8398-4c7ff2248f0c">

However, I think this needs to be tested more before we reopen this issue, as it would definitely cause styling issues (i.e if the size was reduced to fit more characters on screen for example. Closing till the next round of tests.
